models/spring-ai-anthropic/src/main/java/org/springframework/ai/anthropic/api/AnthropicApi.java

```diff
@@ -388,6 +389,7 @@ public record MediaContent( // @formatter:off
 		@JsonProperty("id") String id,
 		@JsonProperty("name") String name,
 		@JsonProperty("input") Map<String, Object> input,
+		@JsonProperty("input_json") String inputJson,
```

Comment:
I understand the reason for this, but It is unfortunate that we need yet another field just to aggregate the partial json.